src/quantize_gptq.py

```diff
@@ -0,0 +1,125 @@
+import argparse
+import logging
+import random
+
+from transformers import AutoTokenizer
+import torch
+import numpy as np
+from datasets import load_dataset
+
+from auto_gptq import AutoGPTQForCausalLM, BaseQuantizeConfig
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)s [%(name)s] %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+def get_dataset(
+    nsamples, seed, seqlen, tokenizer, load_dataset_kwargs, buffer_size=5000
+):
+
+    torch.manual_seed(seed)
```

Comment:
Please use [set_seed in src/utils.py](https://github.com/Nota-NetsPresso/shortened-llm/blob/79e23b49fb7e60b46d68d66856e71bee4574e3f2/src/utils.py#L20-L27).